packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -37,105 +33,69 @@ import transportFetch from './playground-mu-plugin/playground-includes/wp_http_f
 import transportDummy from './playground-mu-plugin/playground-includes/wp_http_dummy.php?raw';
 /* @ts-ignore */
 import playgroundWebMuPlugin from './playground-mu-plugin/0-playground.php?raw';
-import { PHP, PHPResponse, PHPWorker } from '@php-wasm/universal';
+import {
+	PHPResponse,
+	PHPWorker,
+	SupportedPHPVersion,
+	SupportedPHPVersionsList,
```

Comment:
> We should also rename these to say "Minified" instead of "Supported".

This one was mistaken as well. These are supported PHP versions, not minified WP builds.